src/20240130_AaveV3Polygon_FreezeAndSetLTVTo0ForDPIBALCRVAndSUSHIOnAaveV3Polygon20240119/FreezeAndSetLTVTo0ForDPIBALCRVAndSUSHIOnAaveV3Polygon20240119.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Freeze and set LTV to 0 for DPI, BAL, CRV, and SUSHI on Aave v3 Polygon, 2024.01.19"
+author: "Gauntlet"
+discussions: "https://governance.aave.com/t/arfc-recommendation-to-freeze-and-set-ltv-to-0-on-low-cap-aave-v3-polygon-collateral-assets/16311"
+snapshot: https://snapshot.org/#/aave.eth/proposal/0x598f04c1f4fb2acba0a9bdaeb2b18e6e1f43b4c62845ee4b81ae5596a6fc2076
```

Comment:
snapshot only supports a single value

Reply:
Thanks - Do you have any preference for the snapshot link that we include?